static/src/css/modules/_error-page.scss

```diff
@@ -7,11 +8,7 @@
   color: #fff;
   font-family: "Helvetica Neue", "Helvetica", sans-serif;
 
-  a:link, a:visited {
```

Comment:
This was so that we could fix the issue on the not found page where the drop down can't be read
![image](https://github.com/nasa/earthdata-search/assets/34591886/a61fa926-31d9-4c47-9c3c-d2537cc75841)